source/MRViewer/MRWebRequest.h

```diff
@@ -36,6 +42,8 @@ class MRVIEWER_CLASS WebRequest
 
     MRVIEWER_API void setHeaders( std::unordered_map<std::string, std::string> headers );
 
+    MRVIEWER_API void setInputPath( std::string inputPath );
```

Comment:
pleaes add comment as far as it not standart web req field

Reply:
Fixed.